### PR TITLE
made switch tasks

### PR DESCRIPTION
**Pull request by nimdolphin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    break;
+}
+
+// Дана строка, состоящая из символов, например, 'abcde'.
+//  Проверьте, что первым символом этой строки является буква 'a'.
+//  Если это так - выведите 'да', в противном случае выведите 'нет'
+
+let abc = "abcde";
+let symbolA = prompt("введите что-нибудь", "");
+abc = symbolA;
```
Comment (kathrynowy) [inline index.js]: why do you need here 2 variables?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+if (abc[0] === "a") {
+  console.log("yes");
+} else {
+  console.log("no");
+}
```
Comment (kathrynowy) [inline index.js]: do it in 1 line with ternary operator pls (and 1 console.log only)

File: index.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+// Дана строка с цифрами, например, '12345'. Проверьте, что первым
+// символом этой строки является цифра 1, 2 или 3. Если это так -
+//  выведите 'да', в противном случае выведите 'нет'.
+
+let str = "12345";
+let str2 = prompt("введите что-нибудь", "");
+str = str2;
```
Comment (kathrynowy) [inline index.js]: same here

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 3 above)
Comment (nimdolphin) [reply index.js]: fixed

File: index.js (hunk 2 above)
Comment (nimdolphin) [reply index.js]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**